samples/rest/BingAutosuggestV7.py

```diff
@@ -7,31 +7,39 @@
 import os
 import requests
 from pprint import pprint
+from dotenv import load_dotenv
+
+# Load the environment variables from .env file
+load_dotenv()
 
 '''
 This sample uses the Bing Autosuggest API to check the spelling of query words and then suggests corrections.
 Bing Spell Check API: https://docs.microsoft.com/en-us/bing/search-apis/bing-spell-check/overview
 '''
+AUTH_HEADER_NAME='Ocp-Apim-Subscription-Key'
+SUBSCRIPTION_KEY_ENV_VAR_NAME='BING_SEARCH_V7_AUTO_SUGGEST_SUBSCRIPTION_KEY'
 
-# Add your Bing Autosuggest subscription key and endpoint to your environment variables.
-subscription_key = os.environ['BING_AUTOSUGGEST_SUBSCRIPTION_KEY']
-endpoint = os.environ['BING_AUTOSUGGEST_ENDPOINT'] + '/bing/v7.0/Suggestions/'
+# Add your Bing Autosuggest subscription key to your environment variables / .env file
+subscription_key = os.environ.get(SUBSCRIPTION_KEY_ENV_VAR_NAME)
+if subscription_key is None:
+    raise(RuntimeError(f'Please define the {SUBSCRIPTION_KEY_ENV_VAR_NAME} environment variable'))
 
 # Construct the request
-mkt = 'en-US'
+endpoint = 'https://api.bing.microsoft.com/v7.0/suggestions'
 query = 'sail'
+mkt = 'en-US'
 params = { 'q': query, 'mkt': mkt }
-headers = { 'Ocp-Apim-Subscription-Key': subscription_key }
+headers = { AUTH_HEADER_NAME: subscription_key }
 
 # Call the API
 try:
     response = requests.get(endpoint, headers=headers, params=params)
     response.raise_for_status()
 
-    print("\nHeaders:\n")
+    print('\nHeaders:\n')
```

Comment:
I suggest using and IDE linter to autoformat the code style on saving the file after edits.

Reply:
Formatting is reserved for another PR